scala-package/examples/src/main/scala/org/apache/mxnetexamples/rnn/BucketIo.scala

```diff
@@ -94,8 +96,22 @@ object BucketIo {
   class BucketSentenceIter(
       path: String, vocab: Map[String, Int], var buckets: IndexedSeq[Int],
       _batchSize: Int, private val initStates: IndexedSeq[(String, (Int, Int))],
-      seperateChar: String = " <eos> ", text2Id: Text2Id = defaultText2Id,
-      readContent: ReadContent = defaultReadContent) extends DataIter {
+      seperateChar: String, text2Id: Text2Id,
+      readContent: ReadContent,
+      dataLayout: String,
+      labelLayout: String,
+      dataDType : DType,
+      labelDType: DType) extends DataIter {
+
+    // scalastyle:off
```

Comment:
why are turning off scalastyle, i saw in a few other places.

Reply:
Same reason explained above